docker/config/docker_common.env

```diff
@@ -10,9 +10,13 @@
 # specific to the server environment you're setting up.
 
 # alembic
+# -------
```

Comment:
I added "header lines" because this file is big and ornery and getting hard to follow. :(